internal/internal_event_handlers.go

```diff
@@ -818,3 +858,30 @@ func (weh *workflowExecutionEventHandlerImpl) handleChildWorkflowExecutionTermin
 
 	return nil
 }
+
+func (weh *workflowExecutionEventHandlerImpl) handleSignalExternalWorkflowExecutionRequested(event *m.HistoryEvent) error {
+	attributes := event.ExternalWorkflowExecutionSignalRequestedEventAttributes
+	signalID := string(attributes.Control)
+	decision := weh.decisionsHelper.handleSignalExternalWorkflowExecutionRequested(signalID)
+	signal := decision.getData().(*scheduledSignal)
+	if signal.handled {
+		return nil
+	}
+	signal.handle(nil, nil)
```

Comment:
i think you missed the signal data?

Reply:
As discussed, I will just return empty result to user to show success.